DUI3-DX/Converters/Revit/Speckle.Converters.RevitShared/Services/ErrorPreprocessingService.cs

```diff
@@ -0,0 +1,92 @@
+using Autodesk.Revit.DB;
+using Speckle.Core.Logging;
+
+namespace Speckle.Converters.RevitShared.Services;
+
+public class ErrorPreprocessingService : IFailuresPreprocessor
```

Comment:
This seems completely out of scope tbh, we need to design and POC the reporting and this far we've largely been ignoring it. I think this should come out for now - was this a copy and paste?